### PR TITLE
Added deprecation note for score-helm

### PR DESCRIPTION
**Pull request by sujaya-sys**
Added deprecation notice for score-helm

### CODE REVIEW

**Review by mathieu-benoit — APPROVED** [review]
LGTM

File: content/en/docs/score implementation/Other/score-helm.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ aliases:
 - /docs/reference/score-cli/score-helm/
 ---
 
+{{< alert context="info" >}} Deprecation Notice: We have deprecated the score-helm CLI implementation. To get started with Score, we recommend using one of our reference implementations [score-compose](../score-compose.md) or [score-k8s](../score-k8s.md). If you're interested in developing a score-helm reference implementation, we'd love to support you! Please [reach out](https://github.com/score-spec/spec?tab=readme-ov-file#-get-in-touch) to us for assistance and collaboration. {{< /alert >}}
```
Comment (TobiasBabin) [inline content/en/docs/score implementation/Other/score-helm.md:11]: The page links work on GitHub only, for Hugo we need to apply the `relref` type linking:

```suggestion
{{< alert context="info" >}} Deprecation Notice: We have deprecated the score-helm CLI implementation. To get started with Score, we recommend using one of our reference implementations [score-compose]({{< relref "/docs/score implementation/score-compose" >}}) or [score-k8s]({{< relref "/docs/score implementation/score-k8s" >}}). If you're interested in developing a score-helm reference implementation, we'd love to support you! Please [reach out](https://github.com/score-spec/spec?tab=readme-ov-file#-get-in-touch) to us for assistance and collaboration. {{< /alert >}}
```

**Review by TobiasBabin — CHANGES_REQUESTED** [review]
Checking on the local dev server, the links don't work when rendered - please see my commit suggestion.

File: content/en/docs/score implementation/Other/score-helm.md (hunk 1 above)
Comment (TobiasBabin) [reply content/en/docs/score implementation/Other/score-helm.md:11]: Ok looks I was late to the party 🥳 Will prepare another PR to fix the links.

**Pull request merged**